lib/contracts.rb

```diff
@@ -12,6 +12,11 @@
 require "contracts/core"
 
 module Contracts
+  # For backward compatibility
```

Comment:
Actually, this is a shortcut for users who don't want to `include Contracts::Builtin` (because of nameclashes), so that they can type `Contracts::Maybe` or `C::Maybe` (if they have aliased it)